docs/book/src/SUMMARY.md

```diff
@@ -109,7 +109,10 @@
   - [Extending the CLI](./plugins/extending-cli.md)
   - [Creating your own plugins](./plugins/creating-plugins.md)
   - [Available Plugins](./plugins/available-plugins.md)
+    - [go/v3 plugin](./plugins/go-v3-plugin.md)
```

Comment:
This are the guidelines that we should follow up: https://kubernetes.io/docs/contribute/style/style-guide/
If you would like would be great we create a PR to add this one to the contribution guide as well.

Reply:
Hey @camilamacedo86 I didn't  quite understand this comment. Can you please explain it once again?

Reply:
I have raised a PR

Reply:
Sure. It is only for you know.
We ought to follow the K8s style for documentation which is defined in this link

I thought that we did have not this info in our Contribution Guide: https://github.com/kubernetes-sigs/kubebuilder/blob/master/CONTRIBUTING.md . However I just checked that, yes we have: (which is great)

> There are certain writing [style guide](https://kubernetes.io/docs/contribute/style/style-guide/)lines for Kubernetes documentation, the checkout style guide for more information.

Reply:
Okay